src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_STORAGE = 10;
+    private final K[] key;
+    private final V[] value;
+    private int size;
+
+    public StorageImpl() {
+        key = (K[]) new Object[MAX_SIZE_STORAGE];
+        value = (V[]) new Object[MAX_SIZE_STORAGE];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < size; i++) {
+            if ((this.key[i] == key)
```

Comment:
```suggestion
            if ((key[i] == key)
```
and you can remove other "this." in your code. This will improve readability